packages/component-library-vue/src/UtrechtSelect.vue

```diff
@@ -1,17 +1,28 @@
-<script setup lang="ts">
-  defineEmits<(_type: "update:modelValue", _value: unknown) => void>();
+<script lang="ts">
+  import { defineComponent } from "vue";
+  import { useModelWrapper } from "./helpers";
 
-  defineProps<{
-    disabled?: boolean;
-    invalid?: boolean;
-    modelValue?: string;
-    readonly?: boolean;
-    required?: boolean;
-  }>();
+  export default defineComponent({
+    props: {
+      disabled: { type: Boolean, required: false },
+      invalid: { type: Boolean, required: false },
+      readonly: { type: Boolean, required: false },
+      required: { type: Boolean, required: false },
+      options: { type: [Array, Object], required: true },
+      modelValue: { type: [String, Number, Boolean], require: false, default: "" },
```

Comment:
I wonder if we should also invest some time, in the near future, to support generics for `modelValue` so we allow any type here. `defineComponent<T>(` or something, if you know what I mean...

Reply:
I Agree